config/jobs/kubernetes/sig-testing/apidiff.yaml

```diff
@@ -58,10 +58,8 @@ presubmits:
     optional: true
     decorate: true
     annotations:
-      # The apidiff.sh script uses the latest revision of apidiff.
-      # There is no guarantee that this will continue to work for
-      # older branches, so let's not even create per-release
-      # copies of this job.
+      # The script runs for all branches. The behavior of hack/apidiff.sh must be the same
+      # for all branches. If that changes, the job will have to be forked.
```

Comment:
There is no apidiff in release-1.30 at all. Do we need to add it there or do we ignore that it'll fail?

Necessary updates for 1.32 and 1.31:
- https://github.com/kubernetes/kubernetes/pull/129783
- https://github.com/kubernetes/kubernetes/pull/129782

Reply:
We could also exclude running on `release-1.30`.

Reply:
I would skip 1.30, is  End of Life:2025-06-28 and I think we can live with this for 5 months